docs/releases/4.1.txt

```diff
@@ -249,7 +249,10 @@ Upstream support for MariaDB 10.2 ends in May 2022. Django 4.1 supports MariaDB
 Miscellaneous
 -------------
 
-* ...
+* The :djadmin:`test` command returns a non-zero error code for unexpected
+  successes (passing tests marked with :py:func:`unittest.expectedFailure`).
+  Prior to this change, the output would be marked as "FAILED" but the error
+  code was zero, indicating a success.
```

Comment:
Maybe:
```suggestion
* Django test runner now returns returns a non-zero error code for unexpected
  successes from tests marked with :py:func:`unittest.expectedFailure`
  decorator.
```

Reply:
I added a "The" at the beginning of the sentence, removed the "returns" that was there twice, and removed the word "decorator" (I figured people either know what `expectedFailure` is or they can click the link and learn that it's normally used as a decorator).